### ISSUE TITLE
Wrong colors

### ISSUE THREAD

**Issue by ziulev** [Th3Whit3Wolf/one-nvim#9]
I'm using the latest nvim (v0.5.0-dev+1112-ge55ded00c), alacritty with tmux (the result the same without tmux)

init.vim
```
...
syntax on
set termguicolors
set t_Co=256
set background=dark
colorscheme one-nvim
...
```

.zshrc
```
...
export TERM="xterm-256color"
...
```

![image](https://user-images.githubusercontent.com/14956407/112137426-82de5780-8bd0-11eb-82fb-981357f79925.png)

**Comment by rafamadriz** [comment]
I have the exact same problem.

**Comment by Th3Whit3Wolf** [comment]
Are you one the latest commit?

**Comment by rafamadriz** [comment]
I just installed again to test and at first I got an error that basically said one-nvim colorscheme didn't exist. I opened neovim two times and I got the same error. Then when I was going to remove the plugin turns out it now works just fine. 

This is weird, I was testing this theme earlier today I was basically getting the same problem as @ziulev. Also, the error I just got didn't appear earlier today when I installed the theme for the first time. I didn't tweak absolutely nothing to make it work. Don't know why this weird behavior, I installed this with [packer](https://github.com/wbthomason/packer.nvim) doing `use "Th3Whit3Wolf/one-nvim"` maybe something related to that. 

It's working fine now

![2021-Mar-24](https://user-images.githubusercontent.com/67771985/112376908-f8cfe500-8cdc-11eb-932e-2a4532dc66cb.png)
